### PR TITLE
Structure for the To-do List app

### PR DESCRIPTION
**Pull request by kalbek**
On this version of the pull request, simple structure of the To-do App has been built.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
```
Comment (Munsa1) [inline src/index.js:5]: - [ ] In index.js you should create an array of some simple to do tasks (array of objects). Each task object should contain three keys:
```
description [string].
completed [bool].
index: [number].
```
- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task. This should be done dynamically like we did with the project cards on our `Portfolio` projects
e.g
```
The implementation does not have to be exact

const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <div class="list">
+              <input type="checkbox" id="single-list" />
+              <label for="single-list">wash the dishes </label>
+            </div>
+            <i id="selection" class="ptr"></i>
+          </section>
+          <section class="lists">
+            <div class="list">
+              <input type="checkbox" id="single-list" />
+              <label for="single-list">complete To Do list project </label>
```
Comment (Munsa1) [inline src/index.html:36]: - [ ] On page load, the task array should rendered dynamically to the DOM. The tasks should appear in order of the `index` number for each task. Kindly implement this as well

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @kalbek, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Great work on the readme file part :heavy_check_mark: 
Nice job using webpack :heavy_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @kalbek,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Clean Code.
- Items rendered dynamically to the DOM.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**